communication/src/AsyncTlsConnection.cpp

```diff
@@ -469,6 +469,7 @@ std::pair<bool, NodeNum> AsyncTlsConnection::checkCertificate(X509* received_cer
   out.close();
   BIO_free(outbio);
   LOG_INFO(logger_, "new certificate has been updated on local storage, peer: " << peerId);
+  bft::communication::StateControl::instance().restartThinReplicaServer(0);
```

Comment:
How does '0' translates into invoking all callbacks for TRS?

Reply:
Its just added for maintaining the function signature. if we want multiple signatures, we can use variant and remove this unnecessary 0. Adding variant will unnecessary add complexity, hence didn't add. Please suggest your oipnion.